kahuna/public/js/preview/image.js

```diff
@@ -111,13 +111,15 @@ image.controller('uiPreviewImageCtrl', [
       const hasSyndicationUsages$ =
           imageUsagesService.getUsages(ctrl.image).hasSyndicationUsages$;
 
-      const recentUsages$ = imageUsagesService.getUsages(ctrl.image).recentUsages$;
+      const recentPrintUsages$ = imageUsagesService.getUsages(ctrl.image).recentPrintUsages$;
+      const recentDigitalUsages$ = imageUsagesService.getUsages(ctrl.image).recentDigitalUsages$;
```

Comment:
No one else in this file is capturing imageUsagesService.getUsages in a local usages val so presumably there is no performance change from multiple calls to imageUsagesService.getUsages?